src/Generator/TeamSets.php

```diff
@@ -249,7 +249,15 @@ private function addTeamsToCreatedTeamSet($teams)
      */
     private function getRandomArray($array, $num)
     {
-        $rand = array_rand($array, $num);
+        $rand = [];
+        $counterElements = count($array)-1;
+        while (count($rand) < 10) {
+            $indexRandom = mt_rand(0, $counterElements);
+            if (!in_array($indexRandom, $rand)) {
+                $rand[] = $indexRandom;
+            }
+        }
```

Comment:
It turns out that there is already the uniqueness check [here](https://github.com/sugarcrm/Mango/blob/fe1f860b081f5a3a6076bafe0ad11727747096e2/sugarcrm/modules/Teams/TeamSet.php#L200), thus it's fine to have duplicates in the resulting array. Let's remove the duplicate check from this array.